client/components/order-packages/index.js

```diff
@@ -52,3 +51,28 @@ OrderPackages.propTypes = {
 };
 
 export default OrderPackages;
+
+export const Summary = ( { packages, dimensionUnit, weightUnit } ) => {
+	const renderPackageInfo = ( pckg, pckgIndex ) => {
+		return (
+			<li key={ pckgIndex }>
+				{ `${renderPackageDimensions( pckg, dimensionUnit )} (${pckg.weight} ${weightUnit})` }
```

Comment:
Calling `renderPackageDimensions` from a template string seems unnecessary here. Can you store the result in a variable to use instead? I think that would be more easily read.